core_values.py

```diff
@@ -0,0 +1,18 @@
+class CoreValues:
+    def __init__(self):
+        self.values = {}
+
+    def add_value(self, name, weight):
+        self.values[name] = weight
+
+    def remove_value(self, name):
+        if name in self.values:
+            del self.values[name]
```

Comment:
Wouldn't it be useful if it raised an error if the name wasn't in `self.values`?
```suggestion
        if name in self.values:
            del self.values[name]
        else:
            raise KeyError(f"The value '{name}' is not in the list of values")
```